src/main/java/org/spongepowered/common/mixin/entitycollisions/MixinEntity_Collisions.java

```diff
@@ -114,15 +115,16 @@ public void setModDataId(String id) {
     @Override
     public void initializeCollisionState(World worldObj) {
         SpongeConfig<? extends GeneralConfigBase> activeConfig = ((IMixinWorldServer) worldObj).getActiveConfig();
-        EntityCollisionCategory collisionCat = activeConfig.getConfig().getEntityCollisionCategory();
+        SpongeConfig<? extends GeneralConfigBase> globalConfig = SpongeImpl.getGlobalConfig();
+        EntityCollisionCategory collisionCat = globalConfig.getConfig().getEntityCollisionCategory();
         this.maxCollisions = collisionCat.getMaxEntitiesWithinAABB();
```

Comment:
The value should still be read from the `activeConfig` instance, to allow for per-world overrides.